time.go

```diff
@@ -121,19 +122,31 @@ func ParseDateTime(data string) (DateTime, error) {
 	return DateTime{}, lastError
 }
 
-// DateTime is a time but it serializes to ISO8601 format with millis
+// DateTime is a time but it serializes to ISO8601 format with millis.
+//
 // It knows how to read 3 different variations of a RFC3339 date time.
 // Most APIs we encounter want either millisecond or second precision times.
 // This just tries to make it worry-free.
 //
 // swagger:strfmt date-time
 type DateTime time.Time
 
-// NewDateTime is a representation of zero value for DateTime type
+// NewDateTime is a representation of the UNIX epoch (January 1, 1970 00:00:00 UTC) for the DateTime type.
+//
+// Notice that this is not the zero value of the DateTime type.
+//
+// You may use DateTime.IsUNIXZero() to check against this value.
 func NewDateTime() DateTime {
 	return DateTime(time.Unix(0, 0).UTC())
 }
 
+// MakeDateTime is a representation of the zero value of the DateTime type (January 1, year 1, 00:00:00 UTC).
+//
+// You may use Datetime.IsZero() to check against this value.
+func MakeDateTime() DateTime {
+	return DateTime(time.Time{})
+}
```

Comment:
🔗 I would suggest using godoc links here


```suggestion
// NewDateTime is a representation of the UNIX epoch (January 1, 1970 00:00:00 UTC) for the [DateTime] type.
//
// Notice that this is not the zero value of the [DateTime] type.
//
// You may use [DateTime.IsUNIXZero] to check against this value.
func NewDateTime() DateTime {
	return DateTime(time.Unix(0, 0).UTC())
}

// MakeDateTime is a representation of the zero value of the [DateTime] type (January 1, year 1, 00:00:00 UTC).
//
// You may use [Datetime.IsZero] to check against this value.
func MakeDateTime() DateTime {
	return DateTime(time.Time{})
}
```

you can launch `go run golang.org/x/pkgsite/cmd/pkgsite@latest -open` locally to see the difference in rendering

Reply:
cool. Wasn't aware of this option. Adopted